common/xrdp_constants.h

```diff
@@ -284,6 +284,9 @@
 #define RDP_CAPSET_VIRCHAN             20
 #define RDP_CAPLEN_VIRCHAN             0x08
 
+#define RDP_CAPSET_MULTIFRAGMENT       0x001A
```

Comment:
Actually, RDP_CAPSET_SURFCMDS and other constants are much lower in this file (line 576).